components/cli/pkg/commands/list_ingresses.go

```diff
@@ -200,110 +202,164 @@ func displayImageApisTable(cellImageName string) {
 	}
 
 	if cellImageContent.Kind == "Cell" {
-		displayCellImageApisTable(cellImageContent)
+		displayCellImageApisTable(cellImageName)
 	} else if cellImageContent.Kind == "Composite" {
-		displayCompositeImageApisTable(cellImageContent)
+		displayCompositeImageApisTable(cellImageName)
```

Comment:
Should the variable be "cellImageName" since there are both cells and composites? Rename it to "imageName"?

Reply:
Fixed